assets/inject/semver-workflow/lint-staged.template.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   '**/*.{js,jsx}': ['eslint'],
-  '**/*.{ts,tsx}': ['eslint', () => 'tsc --noEmit'],
+  '**/*.{ts,tsx}': ['eslint', () => 'tsc --build'],
```

Comment:
I'm not sure why we want to run type-checking as part of `lint-staged`. Maybe a remnant from the sanipack days?

`tsc --build` is better than `tsc` IMO because it enables incremental mode and supports project references.

I've moved `noEmit: true` to `tsconfig.json`, so it doesn't actually output anything.

Reply:
I like to know that the build compiles when commiting, is the reason. Yeah, it slows down commits.